src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingOptInRequestsOasis.java

```diff
@@ -16,6 +19,7 @@
 
   private static final Page ADS_HOME_PAGE = new Page("project43", "Project43_Wikia");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "project43" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)